exes/ethercat/inc/public/tfc/ec.hpp

```diff
@@ -164,6 +164,16 @@ class context_t {
       return async_start();
     }
 
+    if (config_->value().minimum_slave_count.value().has_value()) {
+      if (slave_count() < config_->value().minimum_slave_count.value().value()) {
+        logger_.trace("Minimum slave count not reached, current slave count: {}, minimum slave count: {}", slave_count(),
+                      config_->value().minimum_slave_count.value().value());
+        return async_start();
+      }
+      logger_.trace("Slave count reached, current slave count: {}, minimum slave count: {}", slave_count(),
+                    config_->value().minimum_slave_count.value().value());
+    }
+
```

Comment:
Could you clean this up a little bit? maybe take a temporary and reference that in the if statements. This is allot of noise to functionality.

Reply:
added variable.